usaspending_api/reporting/v2/views/agencies/overview.py

```diff
@@ -119,6 +119,10 @@ def get_agency_overview(self):
                 "fiscal_year",
                 "fiscal_period",
                 "submission_is_quarter",
+                "unlinked_procurement_c_awards",
+                "unlinked_assistance_c_awards",
+                "unlinked_procurement_d_awards",
+                "unlinked_assistance_d_awards",
             )
             .order_by(
                 f"{'-' if self.pagination.sort_order == 'desc' else ''}{self.pagination.sort_key if self.pagination.sort_key not in ['unlinked_contract_award_count','unlinked_assistance_award_count'] else self.default_sort_column}"
```

Comment:
This line needs to be updated to make sorting functional

Reply:
yep, missed that these are sortable; I fixed & updated ticket AC